StorageSnapshot/ViewModels/DataGridViewModel.cs

```diff
@@ -109,4 +120,29 @@ private void CombineDeviceDetails(Task task)
     public void OnNavigatedFrom()
     {
     }
+
+    public void Receive(UsbDeviceAddedMessage message)
+    {
+        dispatcherQueue.TryEnqueue(async () => {
+            DriveInfo driveInfo = new(message.DeviceId);
+            LocalStorageDevice localStorageDevice = new(driveInfo);
```

Comment:
Again. I wanted to have 1 source to get the LocalStorageDevices from. I was considering the scenario where we had a web service that we had to query for the data. The data always had to come from the service. That is why I tried to keep the creation and maintaining of that data in the .NET Core library. It may not matter but I would at least like to consider having the these view models always pull the LocalStorageDevice's, and the Details, from the .NET Core service. I will spend some time considering creation of these objects outside of the service itself. We could push the creation of the LocalStorageDevices up and require them as parameters to the service when getting details. In that case the app would keep the list but that breaks the client/server pattern I was going for.

Reply:
I completely agree. I didn't enjoy having to update the View separate to the source data. I would have preferred the source data to push changes to the UI instead. 

I believe this to be beyond the scope of this specific issue though.